Dockerfile

```diff
@@ -84,6 +84,5 @@ RUN \
   if test -n "$HLS_VERSION"; then \
     ghcup install hls "$HLS_VERSION" --set; \
     ghcup gc --hls-no-ghc --tmpdirs; \
-    rm --force --recursive --verbose "$CABAL_STORE/*" ~/.cabal/{logs,packages,store}; \
```

Comment:
This wasn't actually deleting anything: https://github.com/acilearning/docker-haskell/actions/runs/6380289724/job/17314394725#step:3:436